root/etc/cont-init.d/50-config

```diff
@@ -357,24 +355,6 @@ else
     echo "Certificate exists; parameters unchanged; starting nginx"
 fi
 
-# create GeoIP2 folder symlink
-[[ -d /var/lib/libmaxminddb ]] && [[ ! -L /var/lib/libmaxminddb ]] && \
-    rm -rf /var/lib/libmaxminddb
-[[ ! -d /var/lib/libmaxminddb ]] && \
-    ln -s /config/geoip2db /var/lib/libmaxminddb
```

Comment:
This symlinks part needs to stay otherwise it will break things for existing users

Reply:
Fixed